components/plan-build-ps1/bin/hab-plan-build.ps1

```diff
@@ -876,7 +877,11 @@ function _Set_DependencyArrays {
   # Build `${pkg_deps_resolved[@]}` containing all resolved direct run
   # dependencies.
   foreach($dep in $pkg_deps) {
-    _install-dependency $dep
+    if ($env:HAB_FEAT_INSTALL_HOOK) {
+      _install-dependency $dep "--ignore-install-hook"
+    } else {
+        _install-dependency $dep
+    }
```

Comment:
That is correct. The idea is that an install hook "does stuff" on whatever node the packaged software is destined to run. Said "stuff" could likely tarnish the state of the machine so I don't think we would ever run it in the build. Now the exception is a dev style studio where you actually do want to run a built service in the studio's supervisor. However we do accomodate that scenario since the Supervisor checks the install hooks in `add_service` (see comments below).